performance-tests/bench_2/worker/main.cpp

```diff
@@ -268,6 +269,79 @@ int ACE_TMAIN(int argc, ACE_TCHAR* argv[]) {
 
     do_wait(config.start_time, "start");
 
+    Log::log() << "Starting Discovery Check." << std::endl;
```

Comment:
Let's move this whole section up a few lines before the "do_wait" for start. The wait for start and the wait for discovery to complete almost fill the same role for most tests, but if we need to wait for some reason after discovery but before we start our test actions (say, for durable data to finish being sent) that's what I envisioned the start_wait being use for.